.github/workflows/tidy-modern.yml

```diff
@@ -0,0 +1,34 @@
+name: tidy-modern
+
+on: [push, pull_request]
+
+jobs:
+  tidy:
+    runs-on: ubuntu-latest
+    container:
+      image: pointcloudlibrary/env:18.04
+    
+    steps:
+    - name: Install git
```

Comment:
The `pointcloudlibrary/env` docker already contains git, so I don't think this is necessary?